packages/webapp/src/containers/Animals/AddAnimals/AddAnimalDetails/usePopulateDetails.ts

```diff
@@ -14,7 +14,7 @@
  */
 import { UseFieldArrayReplace, useFormContext } from 'react-hook-form';
 import { useEffect } from 'react';
-import { groupBy } from 'lodash';
+import groupBy from 'lodash-es/groupBy';
```

Comment:
small thing -- might be good to use format of naming `_groupBy`(lodashgroupBy) it seems to be common probably since the function names are pretty generic and overlappy.